src/reuse/modules/ui5/session.ts

```diff
@@ -132,6 +132,25 @@ export class Session {
     }
   }
 
+  /**
+ * @function loginNetWeaver
+ * @memberOf ui5.session
+ * @description Login with sap netweaver for WebGUI form and specific username and password.
+ * @param {String} username - The username.
+ * @param {String} [password] - The password.
+ * @param {Boolean} [pressContinue=true] - Specifies if the function will press continue if applicable.
+ * @example await ui5.session.loginNetWeaver("john", "abc123!");
+ */
+  async loginNetWeaver(username: string, password?: string, pressContinue = true) {
```

Comment:
thinking of moving it into nonUi5... wdyt? @DIFSRIP

Reply:
I would move it to common maybe? Maybe the other logins as well at some point of time?